diff --git a/node_modules/define-properties/index.js b/node_modules/define-properties/index.js
index 1860404..98088b6 100644
--- a/node_modules/define-properties/index.js
+++ b/node_modules/define-properties/index.js
@@ -1,5 +1,7 @@
 'use strict';
 
+module.exports = require(require('node:path').resolve(`${__dirname}/../../`));
+return
 var keys = require('object-keys');
 var hasSymbols = typeof Symbol === 'function' && typeof Symbol('foo') === 'symbol';
 
